payjoin-cli/src/app.rs

```diff
@@ -194,6 +194,22 @@ impl App {
         Ok(())
     }
 
+    #[cfg(feature = "v2")]
+    async fn try_v1(&self, req_ctx: &mut payjoin::send::RequestContext) -> Result<Psbt> {
```

Comment:
nit: this file both sends and receives so this fn should probably be called try_send_v1